### PR TITLE
#352 Add Strapi Header component into Menu

### PR DESCRIPTION
**Pull request by katarinazaprazna**
- Create a new component `Header` in Strapi
- Embed `Header` into the `Menu` component
- Create a gql fragment for `Header`

The purpose of this PR is enabling the addition of links into navbar header

### CODE REVIEW

File: next/src/services/graphql/queries/menu.gql [hunk 1]
```diff
@@ -1,55 +1,65 @@
...
+        ...MenuSection
+    }
+    seeAllLink {
+        ...Link
+    }
+}
+
+fragment MenuHeader on ComponentHeaderMenuHeader {
+    id
+    links {
```
Comment (katarinazaprazna) [inline next/src/services/graphql/queries/menu.gql:47]: For each link in the header, we should ideally be able to specify its icon name. The links will be rendered as buttons, with each button rendering a specific icon. So far, the only link/button we display is "Kontakty". I assume that for the button's icon, we would hardcode the icon name string. Do we want to add an `iconName` field in Strapi as well? @radoslavzeman

Comment (radoslavzeman) [reply next/src/services/graphql/queries/menu.gql:47]: I would add only one specific `contactsLink` into Strapi.

If there is need for more links, we can implement it later.

File: next/src/services/graphql/queries/menu.gql
Comment (radoslavzeman) [inline next/src/services/graphql/queries/menu.gql:1]: Please fix tab spaces :)

File: strapi/src/components/header-menu/header.json
Comment (radoslavzeman) [inline strapi/src/components/header-menu/header.json:1]: Please move this strapi component into `items` and change the name from `header` to `menu header` (not only display name)

**Review by radoslavzeman — CHANGES_REQUESTED** [review]
Knowing how hard it is to migrate, I want to keep strapi components clean. Please rename the component.

**Review by radoslavzeman — APPROVED** [review]
nice :)

**Pull request merged**